### PR TITLE
Confirmation for stop with visual feedback on request for Job List, Job Detail

### LINKED ISSUE #223: Confirmation for stop, visual feedback on request
### Problem

A "stop" action on a job is irreversible. After the user clicks "stop," they do not receive visual feedback that they have sent a stop request, and they may send additional stop requests.

### Proposed Solution

When the user clicks stop on a job in the list view or details view, ask for confirmation (similar to delete, see #142). If the user confirms the request to stop, display an icon that is not a button to indicate that the stop request has been sent.

### PR DESCRIPTION
**Pull request by andrii-i**
## Description 
- "Stop" icon in `Job List` would now show "Stop" confirmation button on click. On click on "Stop" confirmation button and until page is reloaded, Chip element with "Stopping" text would be shown
- "Stop" button in `Job Detail` page would now show confirmation dialogue before attempting to stop the job. While element is attempting to stop the job (after dialog is confirmed and closed), chip reading "Stopping" would be shown instead of "Stop" button
- Fixes #223 

## Preview
- List view:
list_confirm_stop

- Detail view:
details_confirm_stop

### CODE REVIEW

File: src/mainviews/create-job.tsx [hunk 1]
```diff
@@ -753,7 +753,7 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
     <Box sx={{ p: 4 }}>
       <form className={`${formPrefix}form`} onSubmit={e => e.preventDefault()}>
         <Stack spacing={4}>
-          <Heading level={1}>Create Job</Heading>
+          <Heading level={1}>{trans.__('Create Job')}</Heading>
```
Comment (jweill-aws) [inline src/mainviews/create-job.tsx:756]: This was already fixed in #227; please rebase from `main`

File: src/components/job-row.tsx [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 import { JobFileLink } from './job-file-link';
-
 import { CommandIDs } from '..';
 import { Scheduler } from '../handler';
 import { useTranslator } from '../hooks';
 import { ICreateJobModel } from '../model';
+import DownloadIcon from '@mui/icons-material/Download';
+import StopIcon from '@mui/icons-material/Stop';
+import IconButton from '@mui/material/IconButton';
+import { Stack, TableCell, TableRow } from '@mui/material';
```
Comment (jweill-aws) [inline src/components/job-row.tsx:13]:
```suggestion
import { IconButton, Stack, TableCell, TableRow } from '@mui/material';
```

You can import all the components you need from `@mui/material`.

**Review by jweill-aws — APPROVED** [review]

File: src/components/job-row.tsx (hunk 2 above)
Comment (andrii-i) [reply src/components/job-row.tsx:13]: Thanks. Reminder for me to doublecheck imports, not just 100% rely on auto imports

File: src/components/confirm-icons.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import { Box, Button, Chip, IconButton, SvgIconTypeMap } from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslator } from '../hooks';
+import CloseIcon from '@mui/icons-material/Close';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+
+export function ConfirmIcon(props: {
```
Comment (dlqqq) [inline src/components/confirm-icons.tsx:7]: The names of these components is confusing. `ConfirmIcon` is not an icon but really a button. I suggest the following renames:


```
ConfirmButton => ConfirmDialogButton
ConfirmIcon => ConfirmButton
```

Rename the files accordingly as well.

File: src/components/confirm-icons.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  clickHandler: () => void;
```
Comment (dlqqq) [inline src/components/confirm-icons.tsx:8]: Rename this to `onConfirm`. This really should be `async` as well, but it looks like we're calling stop job via JL commands instead of simply awaiting the scheduler service methods. Refactoring those is outside the scope of this PR.

File: src/components/confirm-buttons.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Chip
+} from '@mui/material';
+import { useTranslator } from '../hooks';
+
+export const ConfirmButton = (props: {
+  handleConfirm: () => Promise<void>;
```
Comment (dlqqq) [inline src/components/confirm-buttons.tsx:15]: Rename to `onConfirm`.

**Review by dlqqq — CHANGES_REQUESTED** [review]
Looks great aside from some naming changes that are needed. Thank you for refactoring our confirmation logic into a separate component!

File: src/components/confirm-buttons.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+        <DialogContent>
+          <DialogContentText>{props.dialogText}</DialogContentText>
+        </DialogContent>
+
+        <DialogActions>
+          <Button
+            variant="contained"
+            //hex of var(--md-grey-600)
+            sx={{ backgroundColor: '#757575' }}
```
[the remark this answers is not in the document]
Comment (andrii-i) [reply src/components/confirm-buttons.tsx:56]: This is much cleaner, thanks

**Review by dlqqq — APPROVED** [review]
